src/Factory/GiiFactory.php

```diff
@@ -6,31 +6,27 @@
 use Psr\Container\ContainerInterface;
 use Yiisoft\Yii\Gii\Gii;
 use Yiisoft\Yii\Gii\GiiInterface;
-use Yiisoft\Yii\Gii\Parameters;
 
 final class GiiFactory
 {
     private array $generators;
+    private array $params = [];
 
-    public function __construct(array $generators = [])
+    public function __construct(array $generators = [], array $params = [])
     {
         $this->generators = $generators;
+        $this->params = $params;
     }
 
     public function __invoke(ContainerInterface $container): GiiInterface
     {
-        $generators = array_merge(
-            $container->get(Parameters::class)->get('gii.generators', []),
-            $this->generators
-        );
         $generatorsInstances = [];
-
-        foreach ($generators as $name => $generator) {
+        foreach ($this->generators as $name => $generator) {
             if (!is_string($name)) {
-                throw new InvalidArgumentException("Generator name must be set.");
+                throw new InvalidArgumentException('Generator name must be set.');
             }
             $generatorsInstances[$name] = $container->get($generator);
```

Comment:
https://github.com/yiisoft/yii-gii/pull/24/files#diff-5a4debdfb8d13de5a8386f362fbdac80R50-R52
Can it be here?

Reply:
Yes.